### PR TITLE
Change geo_columns to dict

### PR DESCRIPTION
**Pull request by adamgilg**
This changes the geo_columns args from lists to dicts in the various transformations.

### CODE REVIEW

**Comment by brandomr** [comment]
This looks good to me. Definitely reminds me we should add some tests for this. @adamgilg if you could send me some example payloads for the various transformations that come from the front end to elwood I could probably crank out tests pretty quickly.

**Comment by brandomr** [comment]
My inclination is to merge this and @ccjoel to deploy it since it's causing a pretty heinous bug when people clip datasets and I feel like they'll continue registering/clipping even though we have this bug and have provided notice about it. @ccjoel thoughts?

**Comment by ccjoel** [comment]
> deploy it since it's causing a pretty heinous bug when people clip datasets and I feel like they'll continue registering/clipping even though we have this bug and have provided notice about it.

Yeah, let me coordinate with Adam today to also merge in the corresponding Dojo PR and we can redeploy the stack.

**Review by ccjoel — CHANGES_REQUESTED** [review]
I took a deeper look on the branch. docstring for the caller of clip_geo (in `elwood/elwood.py`), as an example, still explains that it expects a list instead of a dict. Elwood's top-level [README](https://github.com/jataware/elwood/blob/geo-columns-dict/README.md#geospatial-clipping) also includes mentions of ["lat", "lon"] and should be deplaced with dict examples.

I haven't checked the docs for the rest of the transformations, but we should probably update the docstrings/readme for the rest of the changes to dict as well.

**Comment by ccjoel** [comment]
I'm updating the README now

File: tests/transformations_test.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        """Tear down test fixtures, if any."""
+        gc.collect()
+
+    def test_regrid_dataframe__default(self):
+        """"""
+
+        # TODO
+        # input_path("filename")
+        # regrid_dataframe()
+        assert 3 == 3
```
Comment (ccjoel) [inline tests/transformations_test.py:153]: TODO, I don't have a dataset that allows regridding.

File: elwood/elwood.py [hunk 2]
```diff
@@ -211,30 +211,15 @@ def rescale_dataframe_time(
...
-    """
-
-    return regrid_dataframe(
-        dataframe=dataframe,
-        geo_columns=geo_columns,
-        time_column=time_column,
-        scale_multi=scale_multi,
-        scale=scale,
-    )
+regrid_dataframe_geo = regrid_dataframe
```
Comment (Sorrento110) [inline elwood/elwood.py:214]: If we are doing this on one of the entry points we will probably want to do this on all of the entry-points, but I think this is less human-readable as a library entry-point than what we had previously.

**Review by Sorrento110 — APPROVED** [review]
Looks like good changes to me overall. We can go ahead and merge this and I can publish the new version to pypi.

**Pull request merged**